research/frontend/css/index.css

```diff
@@ -36,6 +41,13 @@ nav {
   padding-top: 30px;
 }
 
+.feed-wrapper {
+  display: flex;
```

Comment:
You'll need to add 'display: -webkit-flex;' for Safari and 'display: -ms-flexbox' for IE suport